GETTING_STARTED.md

```diff
@@ -0,0 +1,215 @@
+# Getting started
+
+In this document you will find a step-by-step guide on how to get Eunomia running in a local Minikube cluster.
```

Comment:
I was also thinking about writing an instruction to get Eunomia running in a Kubernetes cluster in general (Minikube as just an option) but for the sake of simplicity I decided to keep it as it is. If the user already has access to his own Kubernetes cluster configured, I think he'll know exactly what steps to skip to deploy Eunomia to his cluster